### PR TITLE
added via keymap

### PR DESCRIPTION
**Pull request by mankokimochi**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* added new keymap for via support

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Files have been added to the wrong location. Keymaps must be within a folder under `/keymaps/`.

For this case:
```
keyboards/fr4/southpaw75/keymaps/via/keymap.c
keyboards/fr4/southpaw75/keymaps/via/rules.mk
```

File: keyboards/fr4/southpaw75/keymaps/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ),
+    [4] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,               KC_TRNS, KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,          KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,          KC_TRNS,
+        KC_TRNS,            KC_TRNS,             KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (drashna) [inline keyboards/fr4/southpaw75/keymaps/keymap.c:55]: These layers appear empty.   They should be removed, as any empty layers are automatically populated on init.
```suggestion
```

File: keyboards/fr4/southpaw75/keymaps/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (drashna) [inline keyboards/fr4/southpaw75/keymaps/rules.mk:2]:
```suggestion
LTO_ENABLE = yes

```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
Closing as files have still not been moved to the correct location.

**Pull request closed without merging**